src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -18,8 +23,14 @@ public class DeleteCommandParser implements Parser<DeleteCommand> {
      */
     public DeleteCommand parse(String args) throws ParseException {
         try {
-            Index index = ParserUtil.parseIndex(args);
-            return new DeleteCommand(index);
+            args = args.trim();
+            ArrayList<String> stringIndices = new ArrayList<>(new HashSet<String>((Arrays.asList(args.split("\\s+")))));
```

Comment:
Is the purpose of a HashSet here just to remove duplicate indices? Might be good to leave a comment or something for better readability! 

Also minor nit: you could make a `private static final WHITESPACE_SPLIT` to hold the `"\\s+"` regex so it's easier to understand :)